### ISSUE TITLE
Error in training Visa dataset

### ISSUE THREAD

**Issue by yaoyaoyao13** [HuiZhang0812/DiffusionAD#20]
Hello, I downloaded the foregrounds for the Visa dataset in Goole Drive, but the number of images does not match, which caused an error in the training Visa dataset. Could you please upload the file again? Thank you very much

**Comment by HuiZhang0812** [comment]
May I ask which specific subset of the dataset it is?

**Comment by yaoyaoyao13** [comment]
The number of each subclass does not match

**Comment by HuiZhang0812** [comment]
I have reuploaded the foreground of the Visa dataset. You could re-download it again.

**Comment by yaoyaoyao13** [comment]
I downloaded DISthresh from the Visa dataset again, but the issue I raised was not resolved. The images for each subclass were missing, and I asked someone else to download the file, but they did not obtain the complete images.
<img width="550" alt="微信图片_20231219172457" src="https://github.com/HuiZhang0812/DiffusionAD/assets/143946108/ae6c12e1-d670-4878-b758-7f17480e7f77">

**Comment by HuiZhang0812** [comment]
We only extracted the foreground of the data from the training set. I believe the 'missing' IDs you mentioned actually come from the images in the test set.

**Comment by yaoyaoyao13** [comment]
When training the Visa dataset, the training was successfully completed, but this error occurred when reading the mask image. The mask image in the Visa dataset is completely black. Do you know if the author has encountered this error during runtime
<img width="440" alt="微信图片_20231221102015" src="https://github.com/HuiZhang0812/DiffusionAD/assets/143946108/61f10063-ac16-449e-8b22-c75d94075aaf">

**Comment by HuiZhang0812** [comment]
I haven't encountered that before.

**Comment by boxbox2** [comment]
> When training the Visa dataset, the training was successfully completed, but this error occurred when reading the mask image. The mask image in the Visa dataset is completely black. Do you know if the author has encountered this error during runtime <img alt="微信图片_20231221102015" width="440" src="https://private-user-images.githubusercontent.com/143946108/292067934-61f10063-ac16-449e-8b22-c75d94075aaf.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTM3OTYxMjcsIm5iZiI6MTcxMzc5NTgyNywicGF0aCI6Ii8xNDM5NDYxMDgvMjkyMDY3OTM0LTYxZjEwMDYzLWFjMTYtNDQ5ZS04YjIyLWM3NWQ5NDA3NWFhZi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNDIyJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDQyMlQxNDIzNDdaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1iYTRkZmRiMjA1ZGE0ODMxNTZhOTM1ZjBkYWNiMjhmNThhYjA0OWEwODBjN2Y2MTFjZTY2MmI5Zjk0MjlkNDI3JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.aLO-i0S4jpxs2Y2qnSy3fv8wgqHbaOmoeHL0mvGv3CE">

I also encountered this problem. Have you solved it？
